<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWidget</class>
 <widget class="QWidget" name="UpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>130</y>
     <width>470</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>190</y>
     <width>100</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_restart">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重新启动</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_close">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>直接关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="update">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
